inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -146,8 +146,13 @@ def arxiv_plot_extract(obj, eng):
                 return
 
             lb = LiteratureBuilder(source='arxiv', record=obj.data)
+            file_extension_pattern = re.compile('.+(\.[a-zA-Z]+)$')
```

Comment:
Note that there is an official Python way for doing that: `os.path.splitext()`

Reply:
I updated!